test/test_prototype_transforms_kernels.py

```diff
@@ -104,10 +104,7 @@ class TestCommon:
     @pytest.mark.parametrize("device", cpu_and_gpu())
     def test_scripted_vs_eager(self, info, args_kwargs, device):
         kernel_eager = info.kernel
-        try:
-            kernel_scripted = torch.jit.script(kernel_eager)
-        except Exception as error:
-            raise AssertionError("Trying to `torch.jit.script` the kernel raised the error above.") from error
+        kernel_scripted = script(kernel_eager)
```

Comment:
If we already have a cached scripting function, we can also use it here although I locally measured no real difference.